.env.sample

```diff
@@ -2,3 +2,8 @@ DUNE_USER=
 DUNE_PASSWORD=
 DUNE_QUERY_ID=
 FILE_OUT_PATH=./out
+
+# Safe Transaction Service Requirements.
+INFURA_KEY=
+NETWORK=
+PROPOSER_PK=
```

Comment:
Sounds great. I would love to hear the alternative.

Reply:
One alternative I can think of that is probably much more secure is to use an env file that is not directly contained in the project and provide a path to this. This would be similar to storing other types of credentials (like ssh keys). Otherwise I am not aware of there other alternatives. I tried taking a look...

I see some people claiming other file formats for these kinds of things (like JSON or YAML), but not really related to security.

https://support.cloud.engineyard.com/hc/en-us/articles/205407508-Environment-Variables-and-Why-You-Shouldn-t-Use-Them

https://blog.doppler.com/goodbye-env-files

Reply:
We could also use some library to fetch the credentials from 1Password...

https://github.com/1Password/connect-sdk-python